markdown/events/2023/bytesize_crisprseq.md

```diff
@@ -23,3 +25,65 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[13:11](https://www.youtube.com/watch?v=x_eFQW0nNvo&t=791)
+That's the structure of the output folder, I won't go to all the directories in detail, but you will find all the outputs of all the tools used for pre-processing like the join parent reads. It's all the quality filtering steps because we remove sequencing adapters, we remove low quality reads and mask low quality bases also, and then you also have the output of the alignment and finally the most important folder, which is this one called `cigar`. It's called like that because we parse the edits using the cigar field from the mapping. In these directories you will find some tables and summary tables of the edits and also plots.
+
+[14:15](https://www.youtube.com/watch?v=x_eFQW0nNvo&t=855)
+This is an example of the output plots. We report data quality, meaning that you will have a percentage of reads that have good quality, also the ones that were aligned against the reference. We also report the number of reads that were wild type or the ones that contained indels and from these indels we also classify by filter of quality and if they are located in the expected pickup on the cut site and if they are above the sequencing error rate or not. Finally there's also classification between insertions and deletions, if they are insertions produced by our template and also if these indels are in frame or out of frame because the ones that will be out of frame are the more probable to disrupt a gene function and produce a gene knockout.
+
+[15:33](https://www.youtube.com/watch?v=x_eFQW0nNvo&t=933)
+Finally these further steps as Marta already commented that they are already implemented in CRISPR-Analytics and we will add them to crisprseq. This unimolecular clustering step to reduce PCR duplicates or sequencing biases, because usually in the sequencing methodology, shorter reads are sequenced more often but this doesn't mean that you have this particular long deletion more represented in your sample so we can correct with UMIs. Then also the automatic identification of a reference and some noise handling and finally also thinking already about version 2 of crisprseq about the idea that we will be able to analyze other kinds of CRISPR experiments such as CRISPR screening. If you have any doubt or want to work with us, Marta is currently implementing this part of the analysis so you can join the Slack channel and there ask and that's it. Feel free to join this channel, test out the pipeline and see if there's something that you would like to also include. Also check the repository. Thank you.
```

Comment:
```suggestion
Finally these further steps as Marta already commented that they are already implemented in CRISPR-Analytics and we will add them to crisprseq. This unimolecular clustering step to reduce PCR duplicates or sequencing biases, because usually in the sequencing methodology, shorter reads are sequenced more often but this doesn't mean that you have this particular long deletion more represented in your sample so we can correct with UMIs. Then also the automatic identification of a reference and some noise handling and finally also thinking already about version 2 of crisprseq about the idea that we will be able to analyze other kinds of CRISPR experiments such as CRISPR screening. If you have any doubt or want to work with us, Laurence is currently implementing this part of the analysis so you can join the Slack channel and there ask and that's it. Feel free to join this channel, test out the pipeline and see if there's something that you would like to also include. Also check the repository. Thank you.
```
Sorry I dropped this unrelated name, she is another collaborator who will be contribution to the pipeline :D